From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
Date: Wed, 16 Aug 2017 20:17:57 +0300
Subject: [ARCHEOLOGY] [WIP] Initial Rock64 support

> X-Git-Archeology: - Revision 43b3beed8b35cf4ea325068cf5a07dcfd33d98ec: https://github.com/armbian/build/commit/43b3beed8b35cf4ea325068cf5a07dcfd33d98ec
> X-Git-Archeology:   Date: Wed, 16 Aug 2017 20:17:57 +0300
> X-Git-Archeology:   From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
> X-Git-Archeology:   Subject: [WIP] Initial Rock64 support
> X-Git-Archeology:
> X-Git-Archeology: - Revision 9531d1bc7ecd0f468e29e402ba00cbc7b7dd683f: https://github.com/armbian/build/commit/9531d1bc7ecd0f468e29e402ba00cbc7b7dd683f
> X-Git-Archeology:   Date: Sat, 20 Jan 2018 18:58:33 +0300
> X-Git-Archeology:   From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
> X-Git-Archeology:   Subject: Fix rk3328 ATF build configuration
> X-Git-Archeology:
> X-Git-Archeology: - Revision 518ef98107a6705ba6649acdb1481cafedc9766d: https://github.com/armbian/build/commit/518ef98107a6705ba6649acdb1481cafedc9766d
> X-Git-Archeology:   Date: Fri, 16 Feb 2018 19:29:22 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: RK3399. More fixes but Odroid N1 refuses to boot :( ... ATF builds now (after apt install gcc-arm-none-eabi ?). Tested building/booting with 4-byte skip and u-boot with and without loading address. Enough for today.
> X-Git-Archeology:
> X-Git-Archeology: - Revision 1c9ef0872bfdda69e66de11f0e97b6b43218f73c: https://github.com/armbian/build/commit/1c9ef0872bfdda69e66de11f0e97b6b43218f73c
> X-Git-Archeology:   Date: Sun, 24 Nov 2019 22:07:46 +0100
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Multiple bootloader creation options for rk3399 (#1614)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 5a2b2c360b9c26ca4bd0d309af7cd3994fd08b7d: https://github.com/armbian/build/commit/5a2b2c360b9c26ca4bd0d309af7cd3994fd08b7d
> X-Git-Archeology:   Date: Sun, 03 May 2020 19:15:46 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Prepare rockchip64 for switch to mainline u-boot and switched ROCK Pi 4 (#1934)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 91f2700eea18d9886fefa69d54bbbef44682083d: https://github.com/armbian/build/commit/91f2700eea18d9886fefa69d54bbbef44682083d
> X-Git-Archeology:   Date: Wed, 06 May 2020 23:45:09 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Fixes and cleanups after rockchip64 preparation for mainline u-boot (#1945)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0069ad60a6f59028e93cfeebe441538c10738161: https://github.com/armbian/build/commit/0069ad60a6f59028e93cfeebe441538c10738161
> X-Git-Archeology:   Date: Wed, 15 Sep 2021 22:08:39 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Preliminary support for rockchip rk35xx boards (#3111)
> X-Git-Archeology:
> X-Git-Archeology: - Revision de26797423e22d58ec2882d7032c67f77196ecc5: https://github.com/armbian/build/commit/de26797423e22d58ec2882d7032c67f77196ecc5
> X-Git-Archeology:   Date: Sun, 06 Nov 2022 20:32:46 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move all legacy u-boot patches under one general legacy folder (#4386)
---
 trust.ini | 15 ++++++++++
 1 file changed, 15 insertions(+)

diff --git a/trust.ini b/trust.ini
new file mode 100644
index 000000000000..111111111111
--- /dev/null
+++ b/trust.ini
@@ -0,0 +1,15 @@
+[VERSION]
+MAJOR=1
+MINOR=0
+[BL30_OPTION]
+SEC=0
+[BL31_OPTION]
+SEC=0
+PATH=bl31.elf
+ADDR=0x10000
+[BL32_OPTION]
+SEC=0
+[BL33_OPTION]
+SEC=0
+[OUTPUT]
+PATH=trust.bin
-- 
Armbian

